src/Game/GameObjects/Entity.cs

```diff
@@ -70,6 +70,7 @@ protected Entity(uint serial)
         public bool IsClicked;
         public uint LastStepTime;
         public string Name;
+        public int NameCliloc;
```

Comment:
missing the cleanup when pull an entity from the Item/mobile pools

Reply:
👍 should be OK now, only for items since I'm not setting it for mobiles (amended original commit)